Core/StartupArgManager.cs

```diff
@@ -37,5 +36,27 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        public static void MigrateArgsFile()
+        {
+            try
+            {
+                string OldFileLocation = "laargs.txt";
+
+                if (!File.Exists(StartupArgsPath)) { return; }
+                if (string.IsNullOrEmpty(File.ReadAllText(OldFileLocation)) || string.IsNullOrWhiteSpace(File.ReadAllText(OldFileLocation)))
+                    File.Delete(OldFileLocation);
+
+                File.Delete(StartupArgsPath);
+                File.WriteAllText(StartupArgsPath, File.ReadAllText(OldFileLocation));
```

Comment:
This will cause an exception if the old file is empty (and got deleted a couple of lines above). There should be a `return;` inside that if (program returns after deleting an empty old file).